pkg/management/endpoint.go

```diff
@@ -67,102 +68,51 @@ type Endpoints struct {
 	CreateShadowUser endpoint.Endpoint
 }
 
-// ManagementComponent is the interface of the component to send a query to Keycloak.
-type ManagementComponent interface {
```

Comment:
Why is this removed?

Reply:
It is highly linked with Component interface from component.go: if you add a method in one of the two interfaces, you have to add it to the other one. They always have to be exactly duplicated. So it is not required to have both interfaces: we can use only one.